gui/src/components/OnboardingWizard/OnboardingWrapper.tsx

```diff
@@ -0,0 +1,22 @@
+import { useState } from "react";
+
+import { OnboardingWizard } from "./OnboardingWizard";
+
+interface Props {
+  children: React.ReactNode;
+}
+
+export function OnboardingWrapper({ children }: Props) {
+  const onboarded = localStorage.getItem("onboarded");
```

Comment:
This may not be a good idea. As far as I remember, reading from the localStorage is a sync operation; in this case, you do it in every render. You can push this outside of react because you only need to read from the localstorage when you start the app.

Reply:
@naps62 do you think it matters? The localstorage is also persistent

Reply:
Having it in your `settings.json` means you can safely port your settings to a new computer and not have the onboarding jump on you when you already have done it

Reply:
👍, can you help @paulacarneiro make the change?

Reply:
yes. will take care of it tomorrow

Reply:
@paulacarneiro here you go: https://github.com/iron-wallet/iron/pull/322/files

feel free to merge this locally to your branch, or just let me know and I'll merge directly from the PR

with that code, you should be able to use `invoke("settings_get")` (which already existed, you can find it's usage in the code, but it now has a new `onboarded` boolean), and `invoke("settings_finish_onboarding")` to set it to true